tests/unit/acquisition/function/test_function.py

```diff
@@ -264,10 +264,13 @@ def test_mc_expected_improvement_builds_expected_improvement_using_best_from_mod
         tf.constant([[-2.0], [-1.0], [0.0], [1.0], [2.0]]),
         tf.constant([[4.1], [0.9], [0.1], [1.1], [3.9]]),
     )
+    # model = QuadraticMeanAndRBFKernel()
     model = QuadraticMeanAndRBFKernelWithSamplers(dataset)
     acq_fn = MonteCarloExpectedImprovement(int(1e6)).prepare_acquisition_function(model, dataset)
     xs = tf.linspace([[-10.0]], [[10.0]], 100)
     expected = expected_improvement(model, tf.constant([0.0]))(xs)
+    print(expected.shape)
+    print(acq_fn(xs).shape)
```

Comment:
Quick delete this sloppy code before @uri-granta sees

Reply:
Done ;)